roles/test-linux-binary/tasks/main.yml

```diff
@@ -0,0 +1,71 @@
+- name: Compute vars (binary)
+  set_fact:
+    beat_url: "{{url_base}}/{{beat_name}}/{{beat_name}}-{{version}}-{{rpm_arch}}.tar.gz"
+    beat_pkg: "{{beat_name}}-{{version}}-{{rpm_arch}}.tar.gz"
+    beat_cfg: "/tmp/{{beat_name}}-{{version}}-{{rpm_arch}}/{{beat_name}}.yml"
+    workdir: /tmp
+    installdir: /tmp/{{beat_name}}-{{version}}-{{rpm_arch}}
+
+- name: Download package sha1
+  get_url: url={{beat_url}}.sha1.txt dest={{workdir}} validate_certs=no
+
+- name: Download package
+  get_url: url={{beat_url}} dest={{workdir}} validate_certs=no
+
+- name: Check file (linux)
+  shell: chdir={{workdir}} sha1sum -c {{beat_pkg}}.sha1.txt
+
+- name: Untar
+  shell: chdir={{workdir}} tar xzvf {{beat_pkg}}
+
+- name: Disable ES output
+  replace: >
+    dest={{beat_cfg}}
+    regexp='elasticsearch:\n    enabled: true'
+    replace='elasticsearch:\n    enabled: false'
+
+- name: Enable file output (packetbeat)
+  replace: >
+    dest={{beat_cfg}}
+    regexp='#file:\n  #  enabled: true\n  #  path: "/tmp/packetbeat'
+    replace='file:\n    enabled: true\n    path: "{{workdir}}/output'
+  when: beat_name == "packetbeat"
+
+- name: Enable file output (topbeat)
+  replace: >
+    dest={{beat_cfg}}
+    regexp='#file:\n  #  enabled: true\n  #  path: "/tmp/topbeat"\n  #  filename: topbeat'
+    replace='file:\n    enabled: true\n    path: "{{workdir}}/output"\n    filename: topbeat'
+  when: beat_name == "topbeat"
+
+- name: Set Period to 1 second
+  replace: >
+    dest="{{beat_cfg}}"
+    regexp='period: 10'
+    replace='period: 1'
+  when: beat_name == "topbeat"
+
+- name: Start in bg
+  shell: chdir={{installdir}} ./{{beat_name}} -c {{beat_name}}.yml
+  async: 45
+  poll: 0  # run in bg
+
+- name: Sleep a little
```

Comment:
To give Packetbeat a bit of time to initialize before creating the traffic, otherwise it might miss it. Packetbeat is usually quick to start, but it's a race and I did see it fail without the sleep. Would be good to do it some other way, but I couldn't think of a simple solution.